site/en/docs/webstore/using_webstore_api/index.md

```diff
@@ -69,34 +83,39 @@ This will return a result such as:
 ```json
 {
   "access_token" : "ya29...",
-  "token_type" : "Bearer",
   "expires_in" : 3600,
-  "refresh_token" : "1/rwn..."
+  "refresh_token" : "1/rwn...",
+  "scope": "https://www.googleapis.com/auth/chromewebstore",
+  "token_type" : "Bearer",
 }
 ```
 
-You can now use the access_token to call the API. You can also use the refresh token to get future
-access tokens. Note that tokens expire after 40 minutes.
+You can now use the `access_token` to call the API. You can also use the refresh token to get future
+access tokens. Note that tokens expire after an hour.
 
 {% Aside %}
 
-**Note**: For more information about getting OAuth 2.0 access tokens, see [Using OAuth 2.0 to Access
-Google APIs][3].
+For more information about getting OAuth 2.0 access tokens, see [Using OAuth 2.0 to Access Google
+APIs][oauth2].
 
 {% endAside %}
 
 ## Using the API {: #usingtheapi }
 
-Once you have an access token, your app can then use the Chrome Web Store Publish API. There are
-endpoints for creating items, updating items, and publishing items.
+Once you have an access token, your extension can then use the Chrome Web Store Publish API. There
+are endpoints for creating a new item, updating an existing item, and publishing an item. 
 
-{% Aside %}
+Below is a list of considerations for using the Publish API:
 
-**Note**: Currently, there is no API for setting an item's metadata, such as description. This has
-to be done manually in the [Chrome Web Store Developer Dashboard.][4] More detail about the Web
-Store API can be found [here][5].
+- Developers are required to enable Two-Step Verification for their Google Account to publish or
```

Comment:
I don't think 'two-step verification' is a proper noun. In any case, please provide a link that describes how to do this.

Reply:
@dotproto Which of these is the appropriate link?

Reply:
To my knowledge "2-Step Verification" is the prefered term. I think the [How to turn on 2-Step Verification](https://support.google.com/accounts/answer/185839?hl=en&ref_topic=2954345) article is a better reference point than the announcement.

Reply:
Sounds good, let's go with that.